docs/application/toc_all.md

```diff
@@ -237,6 +238,7 @@
 #### [Overview](/application/native/tutorials/details/details.md)
 #### [Application Filtering](/application/native/tutorials/details/app-filtering.md)
 #### [Security and API Privileges](/application/native/tutorials/details/sec-privileges.md)
+#### [Security and API Privileges for Old Apps](/application/native/tutorials/details/old-versioned-sec-privileges.md)
```

Comment:
Suggestion: Old apps does not convey the right meaning. It will be good to specify the version.

Suggest rephrasing to:

[Security and API Privileges for Apps with API Version 4.0 or Earlier](old-versioned-sec-privileges.md)

or

[Security and API Privileges for 4.0 Apps or Earlier](old-versioned-sec-privileges.md)

Please verify and update.

Reply:
done